src/Alac/AlacManipulatorCreator.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace IndieHD\AudioManipulator\Alac;
+
+use IndieHD\AudioManipulator\Converting\ConverterInterface;
+use IndieHD\AudioManipulator\Tagging\TaggerInterface;
+
+class AlacManipulatorCreator implements AlacManipulatorCreatorInterface
```

Comment:
While this class's purpose may not be obvious, the intention is to be able to resolve `AlacManipulator` from the container, albeit indirectly (i.e., through this "proxy" class), given that its constructor requires a primitive argument (in this case, a string that represents a file path).

The use-case is explained very clearly at (this is not my post, FWIW):

https://stackoverflow.com/questions/46084709/how-to-place-a-class-that-requires-a-variable-parameter-into-dependency-injectio

If there's a better means by which to accomplish this, I'm all ears.